include/aspect/assembly.h

```diff
@@ -398,8 +398,8 @@ namespace aspect
         boost::signals2::signal<void (const typename DoFHandler<dim>::active_cell_iterator &,
                                       const double,
                                       const bool,
-                                      internal::Assembly::Scratch::StokesSystem<dim>       &,
-                                      internal::Assembly::CopyData::StokesSystem<dim>      &)> local_assemble_stokes_system;
+                                      internal::Assembly::Scratch::StokesSystem<dim> &,
+                                      internal::Assembly::CopyData::StokesSystem<dim> &)> local_assemble_stokes_system;
```

Comment:
Hi Wolfgang, I'm rather pedantic myself and you've given an excellent reason for keeping the alignment uniform.  Harsha and I will take care of this Friday.  BTW, all of these changes were made by astyle and indent, although I realize at first I was using astyle 2.03.